internal/db/db.go

```diff
@@ -24,9 +24,11 @@ func InitDB() {
 	dbPass := os.Getenv("DB_PASSWORD")
 	dbName := os.Getenv("DB_NAME")
 
-	db, err := sql.Open("postgres", fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=disable", dbHost, dbUser, dbPass, dbName, dbPort))
+	var connErr error
 
-	if err != nil {
+	db, connErr = sql.Open("postgres", fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=disable", dbHost, dbUser, dbPass, dbName, dbPort))
+
+	if connErr != nil {
 		panic(err.Error())
```

Comment:
Every error should just be `err`... not `connErr`... For instance here, the error thrown will be the error returned by `godotenv.Load()` which will be nil leading to panics..

Reply:
I just keep reassigning err?

Reply:
Yep, almost all Go projects I've seen do that